tests/kafkatest/__init__.py

```diff
@@ -22,4 +22,4 @@
 # Instead, in development branches, the version should have a suffix of the form ".devN"
 #
 # For example, when Kafka is at version 1.0.0-SNAPSHOT, this should be something like "1.0.0.dev0"
-__version__ = '6.0.0.dev0'
+__version__ = '6.0.0.dev0'
```

Comment:
We should bump the version to 6.1.0.dev0 here and 6.1.0-SNAPSHOT elsewhere. We now have a 2.6 branch in this repo with version 6.0.0.dev0.

Reply:
Updated the version to 6.1.0.dev0. Updated - 
gradle.properties `version - 6.1.0-ccs-SNAPSHOT`
version.py `DEV_VERSION = KafkaVersion("6.1.0-SNAPSHOT")`